app/presenters/tree_builder_automate_entrypoint.rb

```diff
@@ -1,6 +1,6 @@
 class TreeBuilderAutomateEntrypoint < TreeBuilderAutomateCatalog
   def override(node, _object)
-    node.delete(:selectable)
+    node.selectable = nil
```

Comment:
`false` would be better? (to be consistent with other places)

Reply:
Would be, but it is not what we want. Setting it to `false` means that the node is not selectable, setting it to `nil` means that the value is not set. And if you check the original line, that's exactly what I need.